Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -358,7 +358,9 @@ private TaskDTO convertRelatedJSONObjects(JSONObject jsonObject, TaskDTO taskDTO
         Integer process = getIntegerPropertyForDTO(jsonObject, "process");
         taskDTO.setProcess(serviceManager.getProcessService().findById(process, true));
         Integer processingUser = getIntegerPropertyForDTO(jsonObject, "processingUser");
-        taskDTO.setProcessingUser(serviceManager.getUserService().findById(processingUser, true));
+        if (processingUser != 0) {
```

Comment:
If no user is currently working on this task as he / she aborted this task, this should be reflected into the data.

Reply:
That's exactly what this line does. It checks if some user works on this, if yes, it requests information about this user, if not, it leaves this attribute null.

Reply:
Sorry, I did not see that you are resetting the user. Your code set only the code if the `processingUser` is not `0` and if I look into `getIntegerPropertyForDTO`  it is returning `0` if the value even contains value `null`. So it is never set to an other value even no user is working any more on this task.

Reply:
Could you explain more detailed what is a problem here?

Reply:
I would only ensure that a formally attached user to a task is really removed from indexed data if this user is not involved with this task any more.

Reply:
Method responsible for it is manageProcessingUserDependenciesForIndex(Task task), but I see that there must be actual check if ProcessingUser is not empty.